src/applications/Applications.ExtractImages/ExtractionMessageSender.cs

```diff
@@ -40,8 +41,8 @@ public class ExtractionMessageSender : IExtractionMessageSender
             IProducerModel extractionRequestProducer,
             IProducerModel extractionRequestInfoProducer,
             IFileSystem fileSystem,
-            string extractionRoot,
-            string extractionDir,
+            [NotNull] string? extractionRoot,
+            [NotNull] string? extractionDir,
```

Comment:
I don't think marking these as nullable is correct. We should remove this and any failing tests that raise errors for passing null.